python/py-pyxdg/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pyxdg
+version             0.28
+license             LGPL-2
+categories-append   textproc
+maintainers         {aronnax @lpsinger} openmaintainer
```

Comment:
Please feel free to add yourself as a maintainer.

Reply:
ok